test/integration/GCClassic/integrationTestCreate.sh

```diff
@@ -285,6 +285,13 @@ if [[ "X${testsToRun}" == "XALL" ]]; then
     # 4x5 merra2 fullchem_47L
     create_rundir "1\n1\n1\n1\n2\n${rundirsDir}\n\nn\n" "${log}"
 
+    #=========================================================================
+    # GCAP 2.0 simulations
+    #=========================================================================
+
+    # 2x2.5 ModelE2.1 fullchem (scenario SSP2-4.5, option 6)
+    create_rundir "1\n1\n4\n6\n2\n1\n${rundirsDir}\n\nn\n" "${log}"
+
```

Comment:
FYI, this has already gone into dev/no-diff-to-benchmark.  These updates will conflict with it.

Reply:
Approving with the understanding that @lizziel will rebase to dev/no-diff-to-benchmark.